R/pretty_lollipop_plot.R

```diff
@@ -1,59 +1,219 @@
-#' @title Lollipop Plot
+#' @title Pretty Lollipop Plot.
 #'
-#' @description Generates a visually appealing and interactive lollipop plot.
+#' @description Generates a visually appealing lollipop plot.
 #'
-#' @details This function is depending on a modified version of `readMAF` from the `g3viz` package.
-#' Returned plot is interactive, meaning the user can hover over individual points in the plot to reveal more information.
-#' The plot can also be exported in a variety of file formats inside the interactive view of the lollipop plot.
+#' @details Retrieve meta data of a specific sample or a set of samples. A gene of interest
```

Comment:
Can you please remove here the part about the metadata since the function itself does not handle the metadata? Also, the part about png format is not relevant with the new function setup and can be dropped, too :smile:

Reply:
whoops my bad I missed that!